<!-- xml -->
<launch> 

  <param name="/use_sim_time" value="true"/>
 

  <!-- ROBOT 1 -->
  <group ns="robot_0">
    <param name="tf_prefix" value="robot_0" />
    <param name="amcl/initial_pose_x" value="3" />
    <param name="amcl/initial_pose_y" value="3" />
    <include file="$(find formations)/launch/move_base.launch" >     
      <arg name="robot_name" value="robot_0" />
    </include> 

  </group>

  <group ns="robot_1">
    <param name="tf_prefix" value="robot_1" />
    <param name="amcl/initial_pose_x" value="3" />
    <param name="amcl/initial_pose_y" value="1" />
    <include file="$(find formations)/launch/move_base.launch" >     
      <arg name="robot_name" value="robot_1" />
    </include>    
  </group>
  
  
  <group ns="robot_2">
    <param name="tf_prefix" value="robot_2" />
    <param name="amcl/initial_pose_x" value="1" />
    <param name="amcl/initial_pose_y" value="3" />
    <include file="$(find formations)/launch/move_base.launch" >     
      <arg name="robot_name" value="robot_2" />      
    </include>        
  </group>
  
  <group ns="robot_3">
    <param name="tf_prefix" value="robot_3" />
    <param name="amcl/initial_pose_x" value="1" />
    <param name="amcl/initial_pose_y" value="1" />
    <include file="$(find formations)/launch/move_base.launch" >     
      <arg name="robot_name" value="robot_3" />
    </include>
  </group> 

</launch>
